### DISCUSSION TITLE
希望把时间互转合并到一个页面上

### DISCUSSION THREAD

**Discussion by harryleee** [he3-app/he3-feedback#58] [Ideas]
unix timestamp / format date / utc time ...

开发中会非常频繁的把时间格式转来转去，需要不停的搜素切换页面很繁琐。

参考两个经典产品：
https://tool.chinaz.com/Tools/unixtime.aspx
https://www.unixtimestamp.com/index.php

**Comment by STRRL** [comment]
+1

It would be more powerful if He3 could recognize the format of the timestamp automatically.

**Comment by hongchaodeng** [comment]
你好，He3 的设计理念就是觉得那些“大而全”太复杂了，尽量每一个工具只干一件事。
与其去改工具，我觉得这个其实改进交互的方式会更有效。我们接下来会推出 NLP Chat 方式，在那里你直接把你想要做的转换或者想要使用的工具说出来，然后 He3 会调用相应的工具 :)
